src/common/cockpitwebcertificate.c

```diff
@@ -148,113 +148,3 @@ cockpit_certificate_key_path (const char *certfile)
 
   return keypath;
 }
-
-/**
- * cockpit_certificate_parse:
- *
- * Load the ws certificate file, and split it into the private key and
- * certificates PEM strings.
- *
- * Return 0 on success, or -errno on failure. This can be an error from open()
- * or read(), or ENOKEY if private key is missing.
- */
-int
-cockpit_certificate_parse (const char *file, char **cert, char **key)
```

Comment:
Please also drop the various `PEM_*` constants in this file, and the function declaration in cockpitwebcertificate.h

Reply:
Unresolving -- you still need to drop the declaration from cockpitwebcertificate.h